tensorflow/lite/micro/micro_resource_variable.cc

```diff
@@ -96,9 +97,14 @@ TfLiteStatus MicroResourceVariables::Allocate(int id, TfLiteContext* context,
       MicroPrintf("Failed to allocate resource buffer.");
       return kTfLiteError;
     }
-    // Zero out resource buffers by deafult. Buffers can be initialized to
-    // nonzero values using ASSIGN_VARIABLE.
-    memset(variable.resource_buffer, 0, variable.bytes);
+    // Set resource buffers to the zero_point by default. Buffers can be
```

Comment:
let's add a comment here that link to a specific comment within the bug for more context. That will help the reader understand why this was done. And then we can figure out if zero point is the generic solution or if we want to go in a different direction later.

Reply:
Done.